### ISSUE TITLE
I can't to set attributes

### ISSUE THREAD

**Issue by andresrivas1506** [ging/licode#269]
Hi, I'm trying to set attributes to the stream on my demo, but I am getting this message:
`ERROR: Failed to set attributes data. This Stream object has not been published.`

This is my code:
```javascript
room.addEventListener("stream-added", function (streamEvent) {
      	console.info(streamEvent);
		if (localStream.getID() === streamEvent.stream.getID()) {
		  console.log("Published!!!");
		  console.info('1----------------->',roomID);
		  streamEvent.stream.setAttributes({roomID: roomID});
		  console.info( '------------->', streamEvent.stream.getAttributes().roomID );
		}
        var streams = [];
        streams.push(streamEvent.stream);
        subscribeToStreams(streams);
        document.getElementById("recordButton").disabled = false;
      });
```

And this is my console:
`Object { type="stream-added", stream={...}, msg=undefined}`
`Published!!!`
`1-----------------> 550200b52100a38f05f31fa9`
`ERROR: Failed to set attributes data. This Stream object has not been published.`
`-------------> (cadena vacía)`

What am I doing wrong?

**Comment by nachoaguadoc** [comment]
Hi Andrés,

First of all, there was a bug related to the setAttributes() function. It's already been solved in 96eac07f0d9856de63858ff9e3c1f12772136853. In any case, you have to call setAttributes() in localStream, not in streamEvent.stream. It should be 
```
localStream.setAttributes({roomID: roomID}). 
```
After that, it should work fine

Nacho

**Comment by andresrivas1506** [comment]
Thank you so much, and believe me, I tried all ways possible. Again thanks for fix it!
